CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Introduction
+
+The purpose of this document is to show contributors to the Modernisation Platform code the best practises and testing that should be undertaken. Code should be in Terraform and tests, if coded, in Go.
+
+## Code of Conduct
+
+The Ministry of Justice [Code of Conduct](https://github.com/ministryofjustice/.github/blob/main/CODE_OF_CONDUCT.md) is used to govern the project and those participating.
+Please report unacceptable behavior to the maintainers or administrators.
+
+## Standards
+
+We follow the [MoJ Technical Guidance](https://technical-guidance.service.justice.gov.uk/#moj-technical-guidance),
+which complements the Service Standard,
```

Comment:
Is there a link to the Service Standard? As it's capitalised I presume it's an important document.

Reply:
Copied and pasted from another document. It does bring up a link to the web page when selected based on the section in brackets.

Reply:
I'll amend it to replace the one I copied from the other document